guillotina_elasticsearch/utility.py

```diff
@@ -285,6 +285,7 @@ async def search_raw(
         if "size" in q["body"] and "size" in q:
             # ValueError: Received multiple values for 'size', specify parameters using either body or parameters, not both.
             del q["size"]
+        count = await conn.count(index=index, body={"query": q["body"]["query"]})
```

Comment:
I don't know how expensive is this call but we don't need to do it when `conn.search()` returns less than 10k, right?

Reply:
Yes t'hat's right!